tests/integration_tests/validation/conformance_suite_configurations/edinet.py

```diff
@@ -24,15 +24,6 @@
         "EC5700W.GFM.1.2.13/index.xml:invalid01": {
             "EDINET.EC5700W.GFM.1.1.3": 1,
         },
-        "EC8024E/index.xml:invalid01": {
-            "EDINET.EC8027W": 1,
-        },
-        "EC8058W/index.xml:invalid01": {
-            "EDINET.EC8027W": 1,
-        },
-        "EC8062W/index.xml:invalid01": {
-            "EDINET.EC8027W": 1,
-        },
```

Comment:
Why did these code changes affect EC8027W?

Reply:
I removed fixed the filings to only throw the error that it was meant to throw.